pymfe/_internal.py

```diff
@@ -213,34 +218,34 @@
 """Common exceptions of metafeature extraction."""
 
 
-def warning_format(
-    message: str,
-    category: t.Type[Warning],
-    filename: str,
-    lineno: int,
-    line: str = None,
-) -> str:
-    """Change warnings format to a simpler one.
-
-    Args:
-        message (:obj:`str`): warning message to print.
-
-        category: not used. Just to maintain consistency with warnings API.
-
-        filename: not used. Just to maintain consistency with warnings API.
-
-        lineno: not used. Just to maintain consistency with warnings API.
-
-        line: not used. Just to maintain consistency with warnings API.
-
-    Return:
-        str: formated warning message.
-    """
-    # pylint: disable=W0613
-    return " {} Warning: {}\n".format(VERBOSE_WARNING_SYMBOL, message)
-
-
-warnings.formatwarning = warning_format
+# def warning_format(
```

Comment:
Don't you think it is better to remove the commented codes, or will you use them?

Wasn't it clear to me how warnings will be handled?

Reply:
Maybe it is better to remove this commented code entirely.

The idea is to let the warnings with the default python behavior, since I'm not sure if it is good practice to change the warning format like the way we used to do.